config/environments/production.rb

```diff
@@ -67,9 +71,6 @@
   # Send deprecation notices to registered listeners.
   config.active_support.deprecation = :notify
 
-  # Disable automatic flushing of the log to improve performance.
-  # config.autoflush_log = false
```

Comment:
It's still in rails and defaults to true. It's just not part of the 4.2 production.rb template. btw it's was commented out in the current version.